.github/workflows/nightly.yml

```diff
@@ -920,11 +999,6 @@ jobs:
           docker rmi test-image
           rm -rf app/tomee
 
-      - name: Run Archetype for EE 10 Web Profile, SE 17, WildFly
```

Comment:
Trying to understand why this was removed. Is it redundant?

Reply:
This case was not removed; I simply relocated it to the correct location.

Reply:
I am thinking the diffs may be messed up now. I don’t see a corresponding addition. You may want to start a fresh PR?

Reply:
Sure. #318 will be a continuation of this PR (#310)